Formula/t/tika.rb

```diff
@@ -23,7 +23,7 @@ def install
     bin.write_jar_script libexec/"tika-app-#{version}.jar", "tika"
 
     libexec.install resource("server")
-    bin.write_jar_script libexec/"tika-server-#{version}.jar", "tika-rest-server"
+    bin.write_jar_script libexec/"tika-server-standard-#{version}.jar", "tika-rest-server"
```

Comment:
so this change has been broken since #84739?  cc @joewiz

Reply:
we should probably add server start test for this fix?

(and maybe also add `odie` to sync the versions, they seem in sync)

Reply:
> so this change has been broken since #84739? cc @joewiz

Probably, I didn't check the history. Just needed tika last week and noticed this.